app/workers/update_schedule_worker.rb

```diff
@@ -0,0 +1,10 @@
+class UpdateScheduleWorker
+  include Sidekiq::Worker
+
+  def self.perform
+    Student.all.each do |student|
+      student.update(at_school: false)
+      UpdateScheduleJob.perform(student)
```

Comment:
Well I was thinking that when they sign in it changes the value to true, then every morning at 6 we set it to false. Are you saying we should set it to true every morning? @erinnachen